index.d.ts

```diff
@@ -266,8 +268,14 @@ export class ULabel {
     // Listeners
     public remove_listeners(): void;
     static get_allowed_toolbox_item_enum(): AllowedToolboxItem;
-    static process_classes(ulabel_obj: ULabel, arg1: string, subtask_obj: ULabelSubtask);
-    static build_id_dialogs(ulabel_obj: ULabel);
+    static process_classes(ulabel_obj: ULabel, arg1: string, subtask_obj: ULabelSubtask): void;
+    static build_id_dialogs(ulabel_obj: ULabel): void;
+
+    // Instance init functions
+    public create_overlays(): void;
+
+    // nops
```

Comment:
i may be stupid, what is `nops`?

Reply:
https://en.wikipedia.org/wiki/NOP_(code)

tl;dr `redraw_demo()` doesn't actually do anything